docs/data/material/components/number-field/number-field.md

```diff
@@ -43,7 +47,7 @@ yarn add @base-ui-components/react
 
 ## Outlined field
 
-The outlined field component uses [text-field composition](/material-ui/react-text-field/#components) with end adornments for the increment and decrement buttons.
+The outlined field component uses [`TextField` composition](/material-ui/react-text-field/#components) with end adornments for the increment and decrement buttons.
```

Comment:
@siriwatknp can you elaborate on what this means ("uses text field composition")? I read the section of the doc linked here and it wasn't clear to me how these two are connected.